src/test/java/posters/pageobjects/pages/user/RegisterPage.java

```diff
@@ -38,9 +38,10 @@
 
     @Override
     @Step("ensure this is a register page")
-    public void isExpectedPage()
+    public RegisterPage isExpectedPage()
     {
         registerForm.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.